### DISCUSSION TITLE
A mirror of fastht.ml?

### DISCUSSION THREAD

**Discussion by maphew** [AnswerDotAI/fasthtml#344] [General]
Hi, is there a mirror of *.fastht.ml anywhere? the domain appears to be blocked at our firewall. 

I've asked central IT for why and for it to be whitelisted, but based on past experience this is a long process with an uncertain outcome.

**Comment by akclace** [comment]
For me, fastht.ml does not work over AT&T Fiber. Works fine over mobile and from AWS.

```
$ curl -v https://www.fastht.ml/
*   Trying 104.21.59.237:443...
* Connected to www.fastht.ml (104.21.59.237) port 443
* ALPN: curl offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* Recv failure: Connection reset by peer
* LibreSSL/3.3.6: error:02FFF036:system library:func(4095):Connection reset by peer
* Closing connection
curl: (35) Recv failure: Connection reset by peer
```

I have a proxy running on AWS pointing to fastht.ml. The home page is at https://fasthtml.demo.clace.io and the docs are at https://fasthtml-docs.demo.clace.io/ . The "Read Docs" link on the home page uses an absolute address, so you have to specifically go to the proxied docs page.
